src/account-info/AccountDetails.js

```diff
@@ -76,7 +76,7 @@ export default class AccountDetails extends PureComponent<Props, void> {
         userDetails={() => (
           <View style={styles.details}>
             {status && <UserStatusIndicator status={status} style={styles.statusIndicator} />}
-            <Text style={styles.info}>{email}</Text>
+            <Label style={styles.info} text={email} />
```

Comment:
We are using `Label` for the text which needs to be translated.
https://github.com/zulip/zulip-mobile/blob/8e112271a40135a429c2fcb3eb84fccd14caa712/src/common/Label.js#L26
Email should as it is.